content/modules/reports/pages/configuration.adoc

```diff
@@ -0,0 +1,228 @@
+
+Default value: `1000`.
+--
+
+[[jmix.reports.historyRecordingEnabled]]
+jmix.reports.historyRecordingEnabled::
++
+--
+Activates the xref:run-report.adoc#execution_history[report execution history] mechanism.
+
+Default value: `false`.
+--
+
+[[jmix.reports.officePath]]
+jmix.reports.officePath::
++
+--
+Sets the path to OpenOffice.
+
+Default value: `/`
+--
+
+[[jmix.reports.officePorts]]
+jmix.reports.officePorts::
++
+--
+Specifies the comma- or vertical bar-separated list of available ports for OpenOffice/LibreOffice.
+
+For example: `jmix.reports.officePorts = 8100|8101|8102|8103|8104|8105`.
+
+Default value: `8100, 8101, 8102, 8103`.
+--
+
+[[jmix.reports.pdfFontsDirectory]]
+jmix.reports.pdfFontsDirectory::
++
+--
+Sets to the fonts directory for converting reports from HTML to PDF.
+
+For example: `jmix.reports.pdfFontsDirectory = C:/Windows/Fonts`.
+--
+
+[[jmix.reports.putEmptyRowIfNoDataSelected]]
+jmix.reports.putEmptyRowIfNoDataSelected::
++
+--
+Sets the mode when the bands which datasets have returned no records are still displayed once.
+
+Default value: `true`
+--
+
+[[jmix.reports.saveOutputDocumentsToHistory]]
+jmix.reports.saveOutputDocumentsToHistory::
++
+--
+If set to true, report results files are saved to the xref:files:index.adoc[file store], if the <<jmix.reports.historyRecordingEnabled,jmix.reports.historyRecordingEnabled>> property is on. See xref:run-report.adoc#history_output_documents[Output documents] for details.
+
+Default value: `false`.
+--
+
+[[jmix.reports.useOfficeForDocumentConversion]]
+jmix.reports.useOfficeForDocumentConversion::
++
+--
+Turns on using OpenOffice to convert the report having DOCX template to HTML/PDF, which significantly increases the conversion accuracy.
+
+Default value: `false`
+--
+
+[[jmix.reports.client.backgroundReportProcessingTimeoutMs]]
+jmix.reports.client.backgroundReportProcessingTimeoutMs::
++
+--
+Defines the processing timeout in milliseconds for the report execution if the <<jmix.reports.client.useBackgroundReportProcessing,jmix.reports.client.useBackgroundReportProcessing>> is set to `true`.
+
+Default value: `10000`.
+--
+
+[[jmix.reports.client.enableTabSymbolInDataSetEditor]]
+jmix.reports.client.enableTabSymbolInDataSetEditor::
++
+--
+Defines whether TAB key should be handled as `\t` symbol instead of focus navigation in Script fields of the report editor.
+
+Default value: `false`
+--
+
+[[jmix.reports.client.useBackgroundReportProcessing]]
+jmix.reports.client.useBackgroundReportProcessing::
++
+--
+Allows you to set the report execution as a background task. This property is used in order to add the xref:run-report.adoc#run_cancel[cancel] option.
+
+Default value: `false`.
+--
+
+[[open_office]]
+== Installing and Configuring OpenOffice
```

Comment:
Let's keep only LibreOffice everywhere. It's proven to be much faster in conversion than OpenOffice.